ob-dart.el

```diff
@@ -40,38 +36,34 @@
 ;;     - ob-groovy.el
 ;;     - ob-R.el
 ;;     - ob-python.el
-
+;;
 ;;; Code:
 
 (require 'ob)
-(eval-when-compile (require 'cl))
 
 (defvar org-babel-tangle-lang-exts)
 (add-to-list 'org-babel-tangle-lang-exts '("dart" . "dart"))
 (defvar org-babel-default-header-args:dart '())
-(defvar org-babel-dart-command "dart"
-  "Name of the command to use for executing Dart code. todo - Windows")
+(defvar ob-dart-command "dart"
```

Comment:
Since the package is named `ob-dart`, tooling was insisting that all symbols from this library start with `ob-dart-`.